winui/AvatarView/content-types.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
+description: Learn what are the content types available and how to set value for the content types in WinUI AvatarView control.
```

Comment:
Description: Learn the content types available and how to set a value for the content types in WinUI AvatarView control.